app/controllers/ApprovalController.scala

```diff
@@ -26,7 +26,7 @@ import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 import models.Constants._
 import core.models.errors.{BadRequestError, DuplicateKeyError, Error, NotFoundError, ValidationError, InternalServerError => ServerError}
 import play.api.libs.json.Json.toJson
-
+import Error._
```

Comment:
Should this just have been added to the list for core.models.errors.{} as per other example below?
Just seems odd to import it like this?

Reply:
Its a standard way to import functions so they can used without having to reference the object, however in this instance its no longer needed, its a left over from an early solution to an upgrade issue. I've removed it and pushed the change to the branch.